packages/terra-graphs-docs/src/terra-dev-site/doc/Core/Lifecycle.2.doc.mdx

```diff
@@ -104,4 +106,4 @@ Separation of _container_ and _content_ allows us to:
 -   Draw different combinations of graphs i.e. we can draw a _Combination graph_ with _2 Bar graphs_ or _1 Bar graph_ and _1 line graph_ without ever needing to building those natively in Carbon.
 -   Draw different combinations of graph in any order necessary.
 
-![Alt](../assets/content-interface.png "GraphContent Interface")
+<img src= { graphContent } alt="Graph Content" />
```

Comment:
You might want to scale this image down, it's a bit large.

Reply:
Adjusted as needed [here](https://github.com/cerner/terra-graphs/pull/179/commits/fd0857beac0da90898defb7ab72d686d952409e9#diff-bf4593511b860bc910146c3c868292ec3f99a140967ced3f092b3f0e484b2da6R109).